lib/temporal/client.rb

```diff
@@ -13,26 +13,60 @@ def initialize(config)
       @config = config
     end
 
+    # Starts a workflow with an optional signal.
```

Comment:
Can you please merge this with the `master` branch? I've added YARD-style documentation that should give you some structure for your comments